project/Dependencies.scala

```diff
@@ -1,13 +1,13 @@
 import sbt._
 
 object Dependencies {
-  val akkaV = "2.5.19"
-  val akkaHttpV = "10.1.7"
+  val akkaV = "2.5.31"
+  val akkaHttpV = "10.1.12"
```

Comment:
akka upgrades are not strictly necessary. The workbench-libs upgrade (below), which is necessary, somehow caused missing akka dependencies; upgrading these akka patch versions fixed it, and upgrading seems worthwhile.